dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/archaius/sources/ConfigCenterConfigurationSourceImpl.java

```diff
@@ -147,5 +160,18 @@ public void handle(String action, Map<String, Object> parseConfigs) {
       }
       LOGGER.info("Config value cache changed: action:{}; item:{}", action, configuration.keySet());
     }
+
+    private void replaceConfig(Map<String, Object> configuration, String fileName) {
+      Object tempConfig = configuration.get(fileName);
+      configuration.remove(fileName);
```

Comment:
If remove fileName will cause next configuration update won't get the latest value?